src/almanack/metrics/metrics.yml

```diff
@@ -3,296 +3,354 @@ metrics:
   - name: "repo-path"
     id: "SGA-META-0001"
     result-type: "str"
+    direction: 0
```

Comment:
on it's own (outside this PR), I would be very confused about this key, value pair. Perhaps `sustainability_direction` is a better name? For the value, I'm not sure how to improve it. For example, I don't know how I would interpret `sustainability_direction: 0`.

Reply:
Thanks - good idea, I'll move over to this or something like it within this file.

Reply:
I'm going to try `sustainability_correlation` here instead to help guide this towards clarity.